app/index.js

```diff
@@ -410,7 +410,7 @@ function installDevExtensions(isDev) {
   if (!isDev) {
     return Promise.resolve();
   }
-  const installer = require('electron-devtools-installer'); // eslint-disable-line global-require
+  const installer = require('electron-devtools-installer'); // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Hm. Normally when we have dependencies that are used in both the main process and the renderer we duplicate the dependencies to both `package.json` files. `electron-devtools-installer` is only used in dev however, and is only imported in the `app` itself. The problem is that `install-app-deps` from electron-builder doesn't install devDependencies, so we can't have it there. Not sure what the best solution would be.